tests/tiup-cluster/script/cmd_subtest.sh

```diff
@@ -54,5 +54,11 @@ function cmd_subtest() {
 
     tiup-cluster $client display $name
 
+    tiup-cluster $client --yes clean $name --data --all --ignore-node 172.19.0.102:20160
+
+    echo "checking cleanup data and log"
+    tiup-cluster $client exec $name -N 172.19.0.102 --command "ls /home/tidb/deploy/tikv-20160/log/tikv.log"
```

Comment:
We have a `tiup-cluster _test` hidden sub-command, it's better to add a test for this PR. e.g:
1. insert some rows
2. clean up cluster
3. check the rows